src/auth/auth.service.ts

```diff
@@ -25,7 +25,7 @@ export class AuthService {
 
   async login(user: User): Promise<{ access_token: string }> {
     const tokenPayload: TokenPayload = {
-      sub: user.id,
+      sub: user.userId,
```

Comment:
Why change to user.userId? Its double context.. revert it to id.

Reply:
because I want to put this userId in the chat entity.

Reply:
Ok but still you have double context here. there's no need to type userId, if you already grabbing the id from a user object.

Reply:
lets talk about it I dont understand what you say